### DISCUSSION TITLE
CloverBootloader

### DISCUSSION THREAD

**Discussion by website-admin** [CloverHackyColor/CloverBootloader#433] [Q&A]
Is there a user version of CloverBootloader available for dummies with dinosaur systems?

GA-X58A-UD3R (vers. 2.0) mb with Legacy BIOS only. 

Have Samsung 970 Evo Plus M.2 NVMe SSD on a PCIe (vers. 2) adapter. 

Want to install/boot Windows 7 Professional (SP 1) from the NVMe. (It's an offline workstation using Adobe CS-5, so I can enjoy Aero graphics without worrying about security issues.)

Have Hotfix Standalone updates for NVMe support, but the BIOS doesn't support UEFI.

Need a class 1 UEFI to boot the Legacy system.

TIA for any suggestions.

**Comment by serdeliuk** [comment]
You may want to review CloverV2/BootSectors/Installation.txt file from zip archive to see how to install a provided boot sector to load Clover
